SimulationConfig/inc/SimBookkeeperConfig.hh

```diff
@@ -0,0 +1,35 @@
+#ifndef SimulationConfig_SimBookkeeperConfig_hh
+#define SimulationConfig_SimBookkeeperConfig_hh
+
+//
+// Configuration for the MVACatalog ProditionsEntitiy
+//
+
+#include "fhiclcpp/types/Atom.h"
+#include "fhiclcpp/types/Table.h"
+#include "fhiclcpp/types/Sequence.h"
+
+namespace mu2e {
+
+  struct SimStageEffConfig {
+    using Name=fhicl::Name;
+    using Comment=fhicl::Comment;
+    fhicl::Atom<std::string> tag{
+      Name("tag"), Comment("Identifying tag for this efficiency")};
+    fhicl::Atom<double> eff{
+      Name("eff"), Comment("EFficiency value")};
```

Comment:
It looks like one must specify  stage efficiencies even when the number will be read from a database?
May be I do not understand the intent of the current interface.     I think that the required configuration should contain just a set of "tags".  Then efficiency values for the tags can either be read from the database, or provided via a separate piece of FHICL (which must be absent if DB is being used).

Reply:
I think this is the "separate piece of FHICL" that can be used to override what's in the database. I'm not sure if it has to be absent if the database is being used...